code/_globalvars/configuration.dm

```diff
@@ -8,15 +8,9 @@ GLOBAL_VAR_INIT(game_year, (text2num(time2text(world.realtime, "YYYY")) + 544))
 
 GLOBAL_VAR_INIT(aliens_allowed, 1)
 GLOBAL_VAR_INIT(traitor_scaling, 1)
-//GLOBAL_VAR_INIT(goonsay_allowed, 0)
-GLOBAL_VAR_INIT(dna_ident, 1)
 GLOBAL_VAR_INIT(abandon_allowed, 0)
 GLOBAL_VAR_INIT(enter_allowed, 1)
 GLOBAL_VAR_INIT(guests_allowed, 1)
-GLOBAL_VAR_INIT(shuttle_frozen, 0)
-GLOBAL_VAR_INIT(shuttle_left, 0)
-GLOBAL_VAR_INIT(tinted_weldhelh, 1)
-GLOBAL_VAR_INIT(mouse_respawn_time, 5) //Amount of time that must pass between a player dying as a mouse and repawning as a mouse. In minutes.
```

Comment:
should be a per-mouse class var smh

Reply:
This is honestly better kept as a global rather then a local on the chance that something down the line needs to use it or to make it eaiser to get to later.

Reply:
I really can't see any situation where `mouse_respawn_timer` would ever be needed as a global variable, since it's obviously not used by anything but mice. It could maybe be a config setting, but again I can't imagine it ever needing to be changed.

Reply:
> 
> 
> This is honestly better kept as a global rather then a local on the chance that something down the line needs to use it or to make it eaiser to get to later.

"on the chance that something needs it" isn't a valid excuse.